src/resources/elements/primeDesignSystem/ptoggle/ptoggle.html

```diff
@@ -6,6 +6,7 @@
       value.bind="value"
       class="pToggleInput ${ validationState ? 'pInput-' + validationState : '' }"
       disabled.bind="disabled"
+      change.delegate="changed()"
```

Comment:
This should not be made to look or behave like an event.  It should look and behave like an optional bound property whose type is `(checked: boolean) => void` that the ptoggle viewmodel calls when the component is clicked.